src/tests/Microsoft.PowerFx.Interpreter.Tests/SerializationTests.cs

```diff
@@ -32,7 +33,25 @@ public void DateTimeSerializationTests()
 
                     Assert.Equal(dateTimeValue.Value, dateTimeValueDeserialized.Value);
                 }
-            }            
-        }            
+            }
+        }
+
+        [Fact]
+        public void TempTests()
+        {
+            var engine = new RecalcEngine(new PowerFxConfig());
+
+            var expression = "If(true,";
+
+            var ptBR_opt = new ParserOptions() { Culture = new CultureInfo("pt-BR") };
+            var frFR_opt = new ParserOptions() { Culture = new CultureInfo("fr-FR") };
+            var enGB_opt = new ParserOptions() { Culture = new CultureInfo("en-GB") };
+
+            var ptBR_check = engine.Check(expression, options: ptBR_opt);
+            var frFR_check = engine.Check(expression, options: frFR_opt);
+            var enGB_check = engine.Check(expression, options: enGB_opt);
+
+            Assert.True(ptBR_check.IsSuccess);
```

Comment:
>ptBR_check.IsSuccess); [](http://example.com/codeflow?start=24&length=22)

Success?

Reply:
Just testing a few things. This test case will be changed/removed.